src/tests/mod.rs

```diff
@@ -809,6 +809,40 @@ impl Interface for TestInterface {
     fn save_gas_remaining_before_subexecution(&self, gas_used_until: u64) {
         println!("save_gas_remaining_before_subexecution: {}", gas_used_until);
     }
+
+    fn deferred_call_quote(&self, target_slot: (u64, u8), gas_limit: u64) -> Result<(bool, u64)> {
+        println!(
+            "get_asc_call_fee: target_slot: {:?}, gas_limit: {}",
+            target_slot, gas_limit
+        );
+        Ok((true, 0))
+    }
+
+    fn deferred_call_register(
+        &self,
+        target_slot: (u64, u8),
+        target_addr: &str,
+        target_func: &str,
+        params: &[u8],
+        coins: u64,
+        max_gas: u64,
+    ) -> Result<Vec<u8>> {
```

Comment:
The argument order changed I think (> target_addr, target_func, target_slot, max_gas, params, coins)